OpenTerm/Util/Parsing & Formatting/ANSITextState.swift

```diff
@@ -175,7 +175,7 @@ enum ANSIFontState: Int {
 
 struct ANSITextState {
 	var foregroundColor: UIColor = UserDefaultsController.shared.terminalTextColor
-	var backgroundColor: UIColor = UserDefaultsController.shared.terminalBackgroundColor
+	var backgroundColor: UIColor = .clear
```

Comment:
Now, switching background colors while terminal is open will not keep remnants of the background color visible where text was printed.